bundles/org.openhab.binding.solarman/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -57,6 +57,13 @@
 				<default>60</default>
 				<advanced>true</advanced>
 			</parameter>
+			<parameter name="rawLanMode" type="boolean" required="false">
```

Comment:
Similarly to my comment above, I think a parameter with two "Options" with the Solarman encapsulated protocol as the default, as it seems to be more prevalent,  is more user friendly than a boolean. 

Also it should be described in the [README.md](https://github.com/openhab/openhab-addons/blob/main/bundles/org.openhab.binding.solarman/README.md)

Reply:
I will change this